packages/front-end/components/GeneralSettings/ExperimentSettings/StatsEngineSettings.tsx

```diff
@@ -0,0 +1,399 @@
+  );
+
+  useEffect(() => {
+    if (statsEngineTab === statsEngine) return;
+    setStatsEngineTab(statsEngine);
+    // purposely leave statsEngineTab out of deps so we allow switching tab harmlessly
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [statsEngine]);
+
+  const highlightColor =
+    typeof confidenceLevel !== "undefined"
+      ? confidenceLevel < 70
+        ? "#c73333"
+        : confidenceLevel < 80
+        ? "#e27202"
+        : confidenceLevel < 90
+        ? "#B39F01"
+        : ""
+      : "";
+
+  const warningMsg =
+    typeof confidenceLevel !== "undefined"
+      ? confidenceLevel === 70
+        ? "This is as low as it goes"
+        : confidenceLevel < 75
+        ? "Confidence thresholds this low are not recommended"
+        : confidenceLevel < 80
+        ? "Confidence thresholds this low are not recommended"
+        : confidenceLevel < 90
+        ? "Use caution with values below 90%"
+        : confidenceLevel >= 99
+        ? "Confidence levels 99% and higher can take lots of data to achieve"
+        : ""
+      : "";
+
+  const pHighlightColor =
+    typeof pValueThreshold !== "undefined"
+      ? pValueThreshold > 0.3
+        ? "#c73333"
+        : pValueThreshold > 0.2
+        ? "#e27202"
+        : pValueThreshold > 0.1
+        ? "#B39F01"
+        : ""
+      : "";
+
+  const pWarningMsg =
+    typeof pValueThreshold !== "undefined"
+      ? pValueThreshold === 0.5
+        ? "This is as high as it goes"
+        : pValueThreshold > 0.25
+        ? "P-value thresholds this high are not recommended"
+        : pValueThreshold > 0.2
+        ? "P-value thresholds this high are not recommended"
+        : pValueThreshold > 0.1
+        ? "Use caution with values above 0.1"
+        : pValueThreshold <= 0.01
+        ? "Threshold values of 0.01 and lower can take lots of data to achieve"
+        : ""
+      : "";
+
+  const regressionAdjustmentDaysHighlightColor =
+    typeof regressionAdjustmentDays !== "undefined"
+      ? regressionAdjustmentDays > 28 || regressionAdjustmentDays < 7
+        ? "#e27202"
+        : ""
+      : "";
+
+  const regressionAdjustmentDaysWarningMsg =
+    typeof regressionAdjustmentDays !== "undefined"
+      ? regressionAdjustmentDays > 28
+        ? "Longer lookback periods can sometimes be useful, but also will reduce query performance and may incorporate less useful data"
+        : regressionAdjustmentDays < 7
+        ? "Lookback periods under 7 days tend not to capture enough metric data to reduce variance and may be subject to weekly seasonality"
+        : ""
+      : "";
+
+  return (
+    <div className="mb-3 form-group flex-column align-items-start">
+      <StatsEngineSelect
+        label="Default Statistics Engine"
+        allowUndefined={false}
+        value={form.watch("statsEngine")}
+        onChange={(value) => {
+          setStatsEngineTab(value);
+          form.setValue("statsEngine", value);
```

Comment:
Isn't that redundant with your `useEffect` above?

Reply:
Good catch. The `setStats...` call be removed here. We'll want the useEffect because the form loads async, so on initial render, the tab defaults to 'bayesian'. The useEffect updates the tab when form values are initially loaded or updated. I'll leave a comment to point this out near the useEffect.